index.html

```diff
@@ -41,7 +41,138 @@ <h2 id="title-line">I'am a Software Developer</h2>
+                <ul class="language" id="tech-btn">
+                    <li class="btn-2"><button>html</button></li>
+                    <li class="btn-2"><button>bootstrap</button></li>
+                    <li class="btn-2"><button>Ruby</button></li>
+                </ul>
+                <div class="btn-container">
+                    <button class="btn-3">See Project</button>
+                </div>
+            </div>
+            <div class="projects">
+                <h4 class="header-4">Professional Art Printing Data</h4>
+                <p class="project-details">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                <ul class="language" id="tech-btn">
+                    <li class="btn-2"><button>html</button></li>
+                    <li class="btn-2"><button>bootstrap</button></li>
+                    <li class="btn-2"><button>Ruby</button></li>
+                </ul>
+                <div class="btn-container">
+                    <button class="btn-3">See Project</button>
+                </div>
+            </div>
+        </section> 
+        <section id="about-container">
+            <h5 id="header-5">About me</h5>
+            <p id="about-para">Hello I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.</p>
+            <div id="resume-btn">
+                <button id="btn-resume">Get My Resume</button>
+            </div>
+            <div id="framebg-container">
+                <img src="images/Framebg.png" alt="Graphics Frame" id="frame-bg">
+            </div>
+            <hr id="line-2">
+            <div class="language-header">
+                <img src="images/1.png" alt="" id="heading">
+                <h6 class="header-6">Language</h6>
+            </div>
+            <p class="languages">Javascript</p>
+            <p class="languages">Ruby</p>
+            <p class="languages">Html</p>
+            <p class="languages">Css</p>
+            <div class="language-header">
+                <img src="images/2.png" alt="" id="heading">
+                <h6 class="header-6">Frameworks</h6>
+            </div>
+            <p class="languages">Bootstrap</p>
+            <p class="languages">Ruby on Rails</p>
+            <p class="languages">RSPec</p>
+            <p class="languages">CapyBara</p>
+            <p class="languages">Selenium</p>
+            <div class="language-header">
+                <img src="images/3.png" alt="" id="heading">
+                <h6 class="header-6">Skills</h6>
+            </div>
+            <p class="languages">CodeKit</p>
+            <p class="languages">Github</p>
+            <p class="languages">Codepen</p>
+            <p class="languages">Gitlab</p>
+            <p class="languages">Terminal</p>
+        </section>          
```

Comment:
- [ ] Good job so far 👏 This section looks great but from the markup guidelines, it is expected to be implemented `nested lists` where the technology categories(like `Languages`, `Frameworks`, etc) are supposed to be in list `<ul>`  where each is in `<li>`
but also the technology in each category(e.g: `HTML`, `CSS` etc) is supposed to be in the list `<ul>` where each is in `<li>`
I know it may be tricky to implement it but here is [an article](https://www.webucator.com/article/how-to-properly-nest-lists-in-html/) that will help you understand how to implement a nested list 😉

Reply:
@BertrandConxy  Implemented the said functionality.